CourseGrab/ViewControllers/Home/HomeTableViewController.swift

```diff
@@ -56,6 +57,9 @@ class HomeTableViewController: UITableViewController {
         tableView.register(HomeTableViewHeader.self, forHeaderFooterViewReuseIdentifier: homeHeaderReuseId)
         tableView.register(HomeTableViewCell.self, forCellReuseIdentifier: homeCellReuseId)
 
+        refreshControl = UIRefreshControl()
+        refreshControl?.addTarget(self, action: #selector(refreshTableView), for: .valueChanged)
```

Comment:
Instead of `addTarget` use Tactile’s syntax (we used it in the rest of the app):

    refreshControl.on(.valueChanged, refreshTableView)